integration-tests/taquito-conversion-utils.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { LocalForger } from "@taquito/local-forging";
+import { CONFIGS } from "./config";
+
```

Comment:
wouldn't it make more sense to put these tests inside the already existing integration test for prepare provider? I find the 'taquito conversion utils' aspect of it to be a bit misleading.

we don't really have a set of utilities called that, and the naming also doesn't convey that they're utility functions in prepare provider